cita-executor/core/src/libexecutor/block.rs

```diff
@@ -50,14 +50,14 @@ lazy_static! {
 
 /// Trait for a object that has a state database.
 pub trait Drain {
-    /// Drop this object and return the underlieing database.
-    fn drain(self) -> TrieDB<RocksDB>;
+    /// Drop this object.
+    fn drain(self);
 }
 
 pub struct ExecutedBlock {
     pub block: OpenBlock,
     pub receipts: Vec<Receipt>,
-    pub trie_db: Arc<TrieDB<RocksDB>>,
+    pub state: Arc<RefCell<CitaState<CitaTrieDB>>>,
```

Comment:
Why use RefCell?

Reply:
cita-vm expected a RefCell, and I should build the cell at the beginning of opening an `executedBlock`. 
We should keep only one stateDB for the whole block transactions.

Reply:
> We should keep only one stateDB for the whole block transactions.

one struct? there're three trie: state, tx, receipt,